tests/test_utils.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2018 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA-Workflow-Controller utility tests."""
+
+import io
+import json
+import os
+import uuid
+
+import fs
+import mock
+import pytest
+from flask import url_for
+from pytest_reana.fixtures import (cwl_workflow_with_name,
+                                   cwl_workflow_without_name, default_user,
+                                   sample_yadage_workflow_in_db, session,
+                                   tmp_shared_volume_path,
+                                   yadage_workflow_with_name)
+from reana_db.models import Job, JobCache, Workflow, WorkflowStatus
+from werkzeug.utils import secure_filename
+
+from reana_workflow_controller.errors import WorkflowDeletionError
+from reana_workflow_controller.rest import START, STOP, _delete_workflow
+from reana_workflow_controller.utils import create_workflow_workspace
+
+
+@pytest.mark.parametrize("state", [WorkflowStatus.created,
```

Comment:
Should we use `status` instead of `state`? as we do everywhere else? (sorry I skipped at first)